.vscode/launch.json

```diff
@@ -4,6 +4,20 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        // This configuration is for debugging the currently open test file.
+        {
+            "type": "node",
+            "name": "vscode-jest-tests",
+            "request": "launch",
+            "program": "${workspaceFolder}/node_modules/jest/bin/jest",
+            "args": [
+                "--runInBand",
```

Comment:
That's what the jest add-in recommends for running the tests. I didn't challenge its defaults.

Looking it up, apparently, it [does this](http://jestjs.io/docs/en/cli.html#runinband):

> Run all tests serially in the current process, rather than creating a worker pool of child processes that run tests. This can be useful for debugging.

So, basically, stops multi-threading, which would be a pain when debugging.

Reply:
TIL.  Thanks for looking that up.